projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/utils/block-content.ts

```diff
@@ -63,8 +64,23 @@ type GetTextContentFromBlocksProps = {
  * @returns {GetTextContentFromBlocksProps} The text content.
  */
 
-export function getTextContentFromSelectedBlocks(): GetTextContentFromBlocksProps {
-	const clientIds = select( 'core/block-editor' ).getSelectedBlockClientIds();
+export function useTextContentFromSelectedBlocks(): GetTextContentFromBlocksProps {
```

Comment:
could you move the hook to the hooks folder?

Reply:
Sure. Doing it.

Reply:
Done [7e21505](https://github.com/Automattic/jetpack/pull/31813/commits/7e21505864ec3d315174986e5e6c24d85fa7ae07)